<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleForm</class>
 <widget class="QDialog" name="SaleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить продажу</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="customer_combo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Клиент</string>
    </property>
   </widget>
   <widget class="QComboBox" name="car_combo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Автомобиль</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="datetime_input">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="status_input">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Статус</string>
    </property>
   </widget>
   <widget class="QComboBox" name="employee_combo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Сотрудник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Подтвердить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
